addons/mail/static/src/components/chat_window/chat_window.js

```diff
@@ -237,8 +241,14 @@ class ChatWindow extends Component {
         }
         if (this.chatWindow.isFolded) {
             this.chatWindow.unfold();
+            if (this.chatWindow.threadViewer) {
+                this.chatWindow.threadViewer.addComponentHint('chat-window-unfolded');
+            }
```

Comment:
should probably be part of `unfold` method in model so that other means of unfolding than clicking on header also properly set it